packages/mml-web/src/elements/Video.ts

```diff
@@ -246,6 +254,9 @@ export class Video extends TransformableElement {
         material,
         videoTexture,
       };
+      if (this.props.emissive > 0) {
```

Comment:
I think you can remove this condition to make it clearer that the side-effects on the material can be applied either way